Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -56,15 +67,11 @@ public MixedRealityInputSystemProfile InputSystemProfile
             }
         }
 
-        private IMixedRealityFocusProvider focusProvider = null;
-
         /// <inheritdoc />
-        public IMixedRealityFocusProvider FocusProvider => focusProvider ?? (focusProvider = Registrar.GetService<IMixedRealityFocusProvider>());
-
-        private IMixedRealityRaycastProvider raycastProvider = null;
+        public IMixedRealityFocusProvider FocusProvider => CoreServices.FocusProvider;
 
         /// <inheritdoc />
-        public IMixedRealityRaycastProvider RaycastProvider => raycastProvider ?? (raycastProvider = Registrar.GetService<IMixedRealityRaycastProvider>());
+        public IMixedRealityRaycastProvider RaycastProvider => CoreServices.RaycastProvider;
```

Comment:
Can / should we talk to the registry for these, instead of adding them to `CoreServices`?

Reply:
we could talk to the registry directly. they are treated as core services (constructed and registered by the service locator), hence the change. do you see a reason where this might be the wrong path?